packages/ibm-products/src/components/EmptyStates/EmptyStates.stories.js

```diff
@@ -21,7 +18,7 @@ import { StoryDocsPage } from '../../global/js/utils/StoryDocsPage';
 // import styles from './_storybook-styles.scss';
 
 export default {
-  title: getStoryTitle(EmptyState.displayName),
+  title: 'IBM Products/Patterns/Empty state/EmptyState',
```

Comment:
I can open this as a new issue, but wonder if we might want to group our original EmptyStates together.

Reply:
Sure, we can address it in a follow up PR if that is ok